### PR TITLE
filterXSS is now set up in testSetup.js

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

I came across this while writing tests for #11525 where I had mocked this initially.

## QA Instructions, Screenshots, Recordings

N/A. This is frontend test infrastructure.

### UI accessibility concerns?

N/A

## Added tests?

- [ ] Yes
- [x] No, and this is why: This is test infrastructure.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

A cat sitting on a stool in a coffee shop beside a coffee sitting on a table

### CODE REVIEW

File: app/javascript/readingList/__tests__/readingList.test.jsx [hunk 1]
```diff
@@ -48,7 +48,6 @@ describe('<ReadingList />', () => {
   beforeEach(() => {
     global.fetch = fetch;
     global.getCsrfToken = jest.fn(() => 'this-is-a-csrf-token');
-    global.filterXSS = (text) => text;
```
Comment (nickytonline) [inline app/javascript/readingList/__tests__/readingList.test.jsx:51]: No need to mock this. We can import the real deal.

**Review by mstruve — APPROVED** [review]
🚀

**Review by Zhao-Andy — APPROVED** [review]

File: testSetup.js [hunk 2]
```diff
@@ -1 +1,5 @@
+/* eslint-env node */
 import 'jest-axe/extend-expect';
+import filterXss from './app/assets/javascripts/lib/xss';
+
+global.filterXss = filterXss;
```
Comment (Zhao-Andy) [inline testSetup.js:5]: Wait should this be `global.filterXSS` capital `S`?

**Review by juliannatetreault — APPROVED** [review]
👍 👍

File: testSetup.js (hunk 2 above)
Comment (rhymes) [reply testSetup.js:5]: I think so.

cc @nickytonline: I don't think it's case insensitive, the code calls `filterXSS`, not `filterXss`

Comment (nickytonline) [reply testSetup.js:5]: If you look where we use it in the code it's lower case.

Comment (rhymes) [reply testSetup.js:5]: @nickytonline I'm confused then:

```shell
➜  forem git:(master) rg filterXss
testSetup.js
3:import filterXss from './app/assets/javascripts/lib/xss';
5:global.filterXss = filterXss;
➜  forem git:(master) rg filterXSS | wc -l
      26
```

`filterXss` is only inside `testSetup.js` but `filterXSS` is used 26 times across the website, tests included. Are they two different things?

Comment (nickytonline) [reply testSetup.js:5]: @rhymes, testSetup.js makes it available globally to all tests. Previously it was being mocked in tests and then I realized it didn't need to be.

The import here is literally just for tests.

**Pull request merged**